ui/src/config/section/infra/clusters.js

```diff
@@ -114,8 +114,7 @@ export default {
       message: 'label.outofbandmanagement.enable',
       dataView: true,
       show: (record) => {
-        return record.resourcedetails && record.resourcedetails.outOfBandManagementEnabled &&
-          record.resourcedetails.outOfBandManagementEnabled === 'false'
+        return record.resourcedetails?.outOfBandManagementEnabled === 'false'
```

Comment:
Should be `record?.resourcedetails?.outOfBandManagementEnabled === 'false'`

Reply:
Ok, will apply your suggestion. Thanks for the review.